MTA10/mods/deathmatch/CClient.cpp

```diff
@@ -159,7 +159,17 @@ int CClient::ClientInitialize ( const char* szArguments, CCoreInterface* pCore )
             char* szPass = strtok ( NULL, " " );
 
             // Are we supposed to launch the server and play locally?
-            if ( stricmp ( szArguments, "local" ) == 0 )
+			if (stricmp(szArguments, "play") == 0)
+			{
+				// Create clientgame
+				g_pClientGame = new CClientGame(true);
+

```

Comment:
I also plan to implement functionality to join the previous server connected to and have this behavior the default, then on launch, if the button is pressed, ask the user whether they would like to reconnect to the server they last connected to the last time they had MTA:SA open and if they have never joined a server simply start a local server as-is the case in this commit.